source/docs/software/wpilib-tools/robot-simulation/introduction.rst

```diff
@@ -31,6 +31,11 @@ Ensure the :guilabel:`Desktop Development with C++` option is checked in the Vis
 .. image:: images/vs-build-tools.png
    :alt: Screenshot of the Visual Studio build tools option
 
+Enabling Desktop Support (Python)
+---------------------------------
+
+GUI simulation support is installed when you ``pip install robotpy``.
+
```

Comment:
```
Running Robot Simulation (Python)
------------------------

Robot simulation can be done using the command line, running `python3 -m robotpy sim`.
```
^ Or, link to the robotpy sim instructions from here.

Google search will probably link to here and not to the other robotpy sim instructions